service/round_test.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 var (
-	recoveryExecDecreaseThreshold = 0.95
+	recoveryExecDecreaseThreshold = 0.85
```

Comment:
This is only used in a commented out line. Not sure why it's commented out, but either restore it, or remove the line and the var.